src/client/styles.css

```diff
@@ -796,6 +796,8 @@ body {
     grid-column: 1 / 4;
 }
 
+/* LandingView.js */
```

Comment:
I'm really sorry I changed some of your styling to match the Figma a little more.